launch/judges_server.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+  <node name="judges_server" pkg="ros_interop" type="judges_server.py" output="screen"/>
+  <node name="test_node" pkg="ros_interop" type="testClient.py" output="screen"/>
+</launch>
```

Comment:
Very nit picking but have every file end with a empty line at the end of the file, it will get rid of these red things